### DISCUSSION TITLE
RFC of SaaS Module

### DISCUSSION THREAD

**Discussion by imiskolee** [directus/directus#14115] [Feature Requests]
# What's SaaS Module

Build a public platform to multi clients on the same directus instance.

# What's changes

1. add new table `directus_saas_platforms`
2. Table `directus_users` add column `saas_id`
3. all of the collections have the field `saas_id`(with https://github.com/directus/directus/discussions/14114), and automatically set the value from directus_users.saas_id.
4. permission check add saas_id filter.

**Comment by MichaelFBA** [comment]
Personally I find the permissions setup for multi tenancy in directus to be fairly cumbersome and error prone which is excasabated the larger the app grows. 

We self host with postgres and are currently looking into row level security with possibly setting runtime variables for DB connections. Eg Alternative approach
https://aws.amazon.com/blogs/database/multi-tenant-data-isolation-with-postgresql-row-level-security/

But this is fairly specific because of the underlining DB.

**Reply by imiskolee** [reply]
it's better, but no more databases supports.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
